content/accessibility/links.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Links
+---
+
+## Overview
+Links are user interface elements that *navigate* you to a new place or new content. Contrast this with buttons, which are designed to *activate* a feature. Think of it this way: 
+- Links are transportation vehicles
+- Buttons are the ticket terminals 
+
+## Why?
+Links can do things like help with page context, reference similar items of interest, and allow for endless connected information surfing through Wikipedia. Links play a key part in your experience on the web but without proper consideration those links can be frustrating, skipped over, or completely unnoticed. 
+
+For screen reader assistive technology, links and buttons have expectations to function differently. If a link is activated and does not do what was expected that can be disorienting and frustrating. 
+
+A common way a screen reader might navigate the page is by going through a list of all the links on the page. Without context, "read more" or "click here" links won't be very helpful. 
+
+People who have low or colorblind vision may have trouble identifying links that just use color to distinguish between plain text, this is why keeping the underline styling on links within body text important for identification.
+
+## How to test links
+- Check a link's function. If clicking a link instead activates a feature on the page, it should be a `<button>`, and vice versa. 
+- Check a link's purpose. If you cant get an idea of where a link will take you based on the link text alone, the link text needs updates. 
+- Check a link's contrast. Links being text supersede the widget contrast requirement, which means all links need to pass 4.5:1 against the background color that they are on. 
+- In body text, like a markdown file, check to make sure that all links have underline styling. 
+
+## Guidelines
+
+### For designers
+- Make sure you're using the proper accent color for links in your mocks. Double check the color you're using if you're using it on anything other than a white background. For more context on color, check out our [Color | Interface guidelines (primer.style)](https://primer.style/design/foundations/color).
+- Make sure a link's purpose can be understood on its own without the surrounding context in concise words. 
+- Links should always be underlined if they're in body text. 
+- Links should look like links, not buttons.
```

Comment:
```suggestion
- Links should look like links, not buttons. In rare circumstances, links look like buttons (like calls to action). It's important to make sure that if the action takes you somewhere rather than does something on the page, it's semantically coded as a link regardless of the visual style.
```

Reply:
Yes! 🚀